nccharles/PropertyPro-lite
# #167221844:all endpoints should be validated

### What does this PR do?
**validations**

### Description of Task to be completed?
Endpoints should have validation in order to insert required data
### How should this be manually tested?
git clone this repo
`npm install`
`npm run start-dev`
then open postman to check endpoints validations
**POST** `/api/v1/signup`
**POST** `api/v1/property`

### What are the relevant pivotal tracker stories?
#167221844
